crates/bevy_sprite/src/dynamic_texture_atlas_builder.rs

```diff
@@ -4,19 +4,31 @@ use bevy_math::{IVec2, Rect, Vec2};
 use bevy_render::texture::{Image, TextureFormatPixelInfo};
 use guillotiere::{size2, Allocation, AtlasAllocator};
 
+/// Helper utility to update [`TextureAtlas`] on the fly.
+///
+/// Helpful in cases when texture is created highly dynamic,
```

Comment:
Would this wording be better?

```suggestion
/// Helpful in cases when texture is created procedurally,
```

or
```suggestion
/// Helpful in cases when created texture is highly dynamic,
```